app/components/donation/donation-container.js

```diff
@@ -1,11 +1,24 @@
 import Ember from 'ember';
 
 const {
-  Component
+  Component,
+  computed: {
+    and, empty, or
+  }
 } = Ember;
 
 export default Component.extend({
   classNames: ['donation-container'],
   donationAmount: 0,
-  projectTitle: null
+  projectTitle: null,
+  wasNewCard: true,
+
+  isNewCard: empty('card.id'),
```

Comment:
Someone might wonder whiz this isn't simply `alias(cad.isNew)`, not realizing it's because `card` might be `null` to begin with. Maybe a comment would help?

At least, that's my assumption here. Maybe `card.isNew` would actually work? Does init get called if the component is not rendered?